src/Resources/config/services/twig.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<container xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://symfony.com/schema/dic/services"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <services>
+        <service id="setono_sylius_facebook.twig.extension.events_pushing_statistics"
```

Comment:
Could we make a twig runtime extension? See https://symfony.com/doc/current/templating/twig_extension.html#creating-lazy-loaded-twig-extensions

Reply:
> However, if extensions define lots of complex dependencies (e.g. those making database connections), the performance loss can be significant.

We don't use lots of dependencies :) And the only one dependency we have - don't do any additional connections to database on initialization

Reply:
I just mentioned it because it's best practice ;)